guides/getting-started.md

```diff
@@ -149,13 +149,13 @@ If you don't want to start from one of the recommended samples, you'll need to:
             var identity = new ClaimsIdentity(TokenValidationParameters.DefaultAuthenticationType, Claims.Name, Claims.Role);
 
             // Use the client_id as the subject identifier.
-            identity.AddClaim(Claims.Subject,
+            identity.AddClaim(new Claim(Claims.Subject,
                 await _applicationManager.GetClientIdAsync(application),
-                Destinations.AccessToken, Destinations.IdentityToken);
```

Comment:
It's not a valid change, as the `Claim` constructor doesn't take destinations parameters (which is an OpenIddict-specific concept) but `claimValueType` and `issuer` parameters, that have a totally different meaning.

Instead, consider using the one-shot `identity.SetDestinations()` extension: https://documentation.openiddict.com/guides/migration/30-to-40.html#remove-calls-to-addclaims-that-specify-a-list-of-destinations